### ISSUE TITLE
Bug: Filing item text is cut off after '$' character

### ISSUE THREAD

**Issue by davelacy** [dgunning/edgartools#21]
I have a particular filing and was looking at the items and noticed it cuts off after the dollar sign. This particular example is an 8-K.

```python
# get the company
c = Company('0001588272')

# get the filing
f = c.get_filings().filter(form='8-K')[0]

# display the item text 
print(f.text())
```
Notice the cut-off at:
> which the Company issues shares under its distribution reinvestment plan (the “DRP”) at $

**Comment by dgunning** [comment]
Will take a look

**Comment by dgunning** [comment]
Sorry for the delay. 

**edgartools** uses the **unstructured** library for parsing html and unstructured currently has an issue handling nested div tags.
I'll take a look at how to fix this

**Comment by davelacy** [comment]
No worries at all... not the easiest to parse these things. Thanks for taking a look

**Comment by dgunning** [comment]
Sorry for the long wait. This is fixed in 2.10.1
